chore: add turbo deprecation warning (#132)
stackblitz/docs: docs/platform/webcontainers/turbo-package-manager.md

# {{ $frontmatter.title }}

:::warning
Starting on January 1, 2024, turbo will no longer be supported and will be removed. We now offer built-in support for npm, pnpm, and yarn.

> Adding support for all the features from other package managers into turbo was a lot of work. That's why we decided to focus on supporting the native package managers out of the box instead of building our own.

Author: Yeah maybe not. But if we are linking here from the terminal deprecation warning, I guess we want a little more context? Not really sure what to put here though.

> Adding support for all the features from other package managers into turbo was a lot of work. That's why we decided to focus on supporting the native package managers out of the box instead of building our own.

Reviewer: ```suggestion
Adding support for all the features from other package managers into turbo required a lot of continuous work. Because of that, we now support native package managers instead of building our own.
```
